storage-access.bs

```diff
@@ -162,6 +176,8 @@ To <dfn type="abstract-op">save the storage access flag set</dfn> for a [=partit
 partial interface Document {
   Promise&lt;boolean> hasStorageAccess();
   Promise&lt;undefined> requestStorageAccess();
+  Promise&lt;undefined> allowStorageAccessRequestOnSite(DOMString serializedSite);
+  Promise&lt;undefined> requestStorageAccessUnderSite(DOMString serializedSite);
```

Comment:
I'm not sure what typical browser parlance is used for APIs like this, but I would think "origin" might be better than "site"?

```suggestion
  Promise&lt;undefined> allowStorageAccessRequestOnOrigin(DOMString serializedSite);
  Promise&lt;undefined> requestStorageAccessUnderOrigin(DOMString serializedSite);
```

Reply:
This API is specifically Site-scoped, using [the definition of site from whatwg](https://html.spec.whatwg.org/multipage/origin.html#site).